README.md

```diff
@@ -63,3 +64,38 @@ In order to run the full suite of Acceptance tests, run `make testacc`.
 ```sh
 $ make testacc
 ```
+
+Acceptance test prerequisites
+-----------------------------
+In order to successfully run the full suite of acceptance tests, you will need to have the following:
+
+### GitHub personal access token
+You will need to create a [personal access token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) for
+testing. It will need to have the following scopes selected:
+* repo
+* admin:org
+* admin:public_key
+* admin:repo_hook
+* admin:org_hook
+* user
+* delete_repo
+* admin:gpg_key
+
+Once the token has been created, it must be exported in your environment as `GITHUB_TOKEN`.
+
+### GitHub organization
+If you do not have an organization already that you are comfortable running tests again, you will need to [create one](https://help.github.com/en/articles/creating-a-new-organization-from-scratch). The free "Team for Open Source" org type is fine for these tests. The name of the
+organization must then be exported in your environment as `GITHUB_ORGANIZATION`.
+
+### Test repository
```

Comment:
Definitely not for this pull request, but we should fix the testing that requires this existing repository to instead automatically create it so this step is not required. 👍

Seems like its related to the `github_repository` data source testing and could be fixed via something like:

```go
func testAccCheckGithubRepositoryDataSourceConfig_fullName(randomName string) string {
	return fmt.Sprintf(`
resource "github_repository" "test" {
  name = %[1]q
  # ... appropriate configuration ...
}

data "github_repository" "test" {
  full_name = "${github_repository.test.full_name}"
}
`, randomName)
}

func testAccCheckGithubRepositoryDataSourceConfig_name(randomName string) string {
	return fmt.Sprintf(`
resource "github_repository" "test" {
  name = %[1]q
  # ... appropriate configuration ...
}

data "github_repository" "test" {
  name = "${github_repository.test.name}"
}
`, randomName)
}
```